README.md

```diff
@@ -6,6 +6,8 @@ Particula is a simple, fast, and powerful particle simulator, or at least two of
 
 The main goal is to develop an aerosol particle model that is usable, efficient, and productive. In this process, we all will learn developing models in Python and associated packages. Let us all be friendly, respectful, and nice to each other. Any code added to this repository is automatically owned by all. Please speak up if something (even if trivial) bothers you. Talking through things always helps. This is an open source project, so feel free to contribute, however small or big your contribution may be.
 
+We follow the Google Python style guide [here](https://google.github.io/styleguide/pyguide.html).
+
```

Comment:
Good job on the additional line there (10), otherwise markdown-linter would have annoyed me to fix it!